midi_app_controller/gui/binds_editor.py

```diff
@@ -364,85 +378,92 @@ def __init__(
         """
         super().__init__()
 
+        self.setWindowTitle("Edit Binds")
         self.binds = binds.copy(deep=True)
         self.save_binds = save_binds
 
+        name_label = QLabel("Name:")
         self.name_edit = QLineEdit(binds.name)
+        name_layout = QHBoxLayout()
+        name_layout.addWidget(name_label)
+        name_layout.addWidget(self.name_edit)
+
+        self.show_action_names_checkbox_button = QCheckBox("Show action names")
+        self.show_action_names_checkbox_button.setChecked(False)
+        self.show_action_names_checkbox_button.stateChanged.connect(
+            self._toggle_names_mode
+        )
+
+        self.show_action_names_checkbox_knob = QCheckBox("Show action names")
+        self.show_action_names_checkbox_knob.setChecked(False)
+        self.show_action_names_checkbox_knob.stateChanged.connect(
+            self._toggle_names_mode
+        )
 
         # Save/exit buttons.
-        toggle_names_mode_button = QPushButton("Toggle names mode")
-        toggle_names_mode_button.clicked.connect(self._toggle_names_mode)
         save_and_exit_button = QPushButton("Save and exit")
         save_and_exit_button.clicked.connect(self._save_and_exit)
         exit_button = QPushButton("Exit")
-        exit_button.clicked.connect(self._exit)
+        exit_button.clicked.connect(self.close)
```

Comment:
```suggestion
        exit_button.clicked.connect(self._exit)
```
There is (not on main yet) some code in `self._exit` that waits for threads etc.